libindy/src/errors/wallet.rs

```diff
@@ -134,7 +134,7 @@ impl ToErrorCode for WalletError {
             WalletError::ItemNotFound => ErrorCode::WalletItemNotFound,
             WalletError::ItemAlreadyExists => ErrorCode::WalletItemAlreadyExists,
             WalletError::QueryError(_) => ErrorCode::WalletQueryError,
-            WalletError::NotClosed => ErrorCode::WalletNotClosed,
+            WalletError::NotClosed => ErrorCode::CommonInvalidState,
```

Comment:
I suggest to remove WalletError::NotClosed at all, not just use different error code mapping. Mapping of errors should be one to one.